src/Umbraco.Cms.Integrations.Crm.Dynamics/Controllers/FormsController.cs

```diff
@@ -96,20 +96,24 @@ public async Task<string> GetAccessToken([FromBody] OAuthRequestDto authRequestD
                 { "code", authRequestDto.Code }
             };
 
-            var requestMessage = new HttpRequestMessage
+            string responseContent = string.Empty;
+            try
```

Comment:
This looks fine, but I'm a bit surprised it means you have to change the pattern used previously.  Could we not do a `if (response.IsSuccessStatusCode)` like we did before.

And then if that is `false`, examine: `if (response.StatusCode == HttpStatusCode.Unauthorized)`

Reply:
Updated